### ISSUE TITLE
Compatibility with next 15?

### ISSUE THREAD

**Issue by j2is** [shuding/next-view-transitions#42]
I'm using next 15.0.2, would be nice to have this work with this

**Comment by shuding** [comment]
Haven't looked into it and unsure if there's anything block, would appreciate any PRs!

**Comment by pietrodev07** [comment]
> Haven't looked into it and unsure if there's anything blocking, would appreciate any PRs!

@shuding I'm using next 15 and it works well, you just need to upgrade the library dependencies if you don't want to get a peer dependencies warning!!

You can modify the package.json like this:

```json
"peerDependencies": {
  "next": ">=14.0.0",
  "react": ">=18.2.0",
  "react-dom": ">=18.2.0"
}
```

**Comment by j2is** [comment]
I made a pull request, but may be just check it before integrating? Thanks so much

**Comment by shuding** [comment]
Fixed with #50.

**Comment by ManuLpz4** [comment]
> Fixed with [#50](https://github.com/shuding/next-view-transitions/pull/50).

Not working without flags
